iot-hub/Samples/device/EdgeDownstreamDevice/Program.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography.X509Certificates;
+
+using Microsoft.Azure.Devices.Client;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    class Program
+    {
+
+        // 1) Obtain the connection string for your downstream device and to it
```

Comment:
This is complicated to understand. I think the sample should not be available here: instead this should be an iotedge sample containing both the edge setup _and_ this deviceclient sample.
Ideally, you would be able to use the existing sample and only modify the connection string right?